telegram/ext/filters.py

```diff
@@ -324,6 +324,65 @@ def filter(self, message):
 
     group = _Group()
 
+    class user(BaseFilter):
```

Comment:
Sorry for the delay,  just saw your replies here.

Uhm. I'm not sure.
In contrary to other fillers we provide, this filter still needs to be instantiated and cannot be used otherwise.